core/src/main/java/com/datastax/oss/driver/api/core/cql/Statement.java

```diff
@@ -348,6 +348,20 @@ default SelfT setRoutingKey(@NonNull ByteBuffer... newRoutingKeyComponents) {
   /** Whether tracing information should be recorded for this statement. */
   boolean isTracing();
 
+  /**
+   * Sets the "now in seconds" to use when applying the request (for testing purposes). {@link
+   * Integer#MIN_VALUE} means "no value".
+   *
+   * <p>This method's default implementation returns the statement unchanged. The only reason it
+   * exists is to preserve binary compatibility. Internally, the driver overrides it to record the
+   * new value.
+   */
+  @NonNull
+  @SuppressWarnings("unchecked")
+  default SelfT setNowInSeconds(int nowInSeconds) {
+    return (SelfT) this;
+  }
```

Comment:
Since this feature is mainly intended for tests, I think default implementations are a good compromise to avoid breaking the API.